pkg/ecdsa/retry_test.go

```diff
@@ -0,0 +1,106 @@
+package ecdsa
```

Comment:
Can we add a scenario covering the unhappy path when `int(quantity) > len(groupMembers)`?

Reply:
[0b2fb77](https://github.com/keep-network/keep-core/pull/3109/commits/0b2fb77e7f41b5ecfbb1a957cb94b3955f884412
)